tests/data/claypit/tests/math2/MathCest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use \Math\CalcHelper as Calc;
+
+class MathCest
```

Comment:
Builds are failing with

> Fatal error: Cannot redeclare class MathCest in /home/travis/build/Codeception/Codeception/tests/data/claypit/tests/math2/MathCest.php on line 6

Rename it to Math2Cest.

Reply:
I just updated the branch. Is there any change this change is being merged into the 3.x branch as well? I think a lot of us still use version 3. Without this fix it would be impossible to switch to pcov and not lose code coverage metrics.